Quick update to Fleet Server main paragraph (#621)
elastic/observability-docs: docs/en/ingest-management/fleet/fleet-server.asciidoc

beta[]

> {fleet-server} is a component in the {stack} used to centrally manage {agent}s.
> It is launched as part of an {agent} on a host intended to act as this server.
> One {fleet-server} process can support many Agents {agent} connections.  Further,
> {fleet-server} serves as a control plane for updating agent policies, collecting
> status information, and coordinating actions across {agent}s.

Reviewer: ```suggestion
{fleet-server} is a component of the {stack} used to centrally manage {agent}s.
It's launched as part of an {agent} on a host intended to act as a server.
One {fleet-server} process can support many {agent} connections,
and serves as a control plane for updating agent policies, collecting
status information, and coordinating actions across {agent}s.
```

Author: fabulous - thanks Brandon!
